bridge/src/ncg-transfer.ts

```diff
@@ -0,0 +1,65 @@
+import axios, { AxiosResponse } from "axios";
+import { INCGTransfer } from "./interfaces/ncg-transfer";
+import { TxId } from "./types/txid";
+
+interface GraphQLRequestBody {
+    operationName: string | null;
+    query: string;
+    variables: Object;
+}
+
+export class NCGTransfer implements INCGTransfer {
+    private readonly _apiEndpoint: string;
+
+    /**
+     * The address of sender;
+     */
+    private readonly _address: string;
+
+    constructor(apiEndpoint: string, address: string) {
+        this._apiEndpoint = apiEndpoint;
+        this._address = address;
+    }
+
+    async transfer(address: string, amount: BigInt): Promise<TxId> {
+        const nextTxNonce = await this.getNextTxNonce(this._address);
+        const query = `mutation TransferGold($recipient: Address!, $amount: String!, $txNonce: Long!)
+        { transfer(recipient: $recipient, amount: $amount, txNonce: $txNonce) }`;
+        const response = await this.graphqlRequest({
+            operationName: "TransferGold",
+            query,
+            variables: {
+                recipient: address,
+                amount: amount.toString(),
+                txNonce: nextTxNonce,
+            }});
+
+        console.log(response);
+        console.log(response.data);
+        console.log(response.data.errors);
```

Comment:
Are these logs still needed?

Reply:
No, I used it for debugging, it will remove them from commits. Thanks 🙏🏻

Reply:
I applied it in new commits. Could you review them again?